Update URL for Debian cloud images (#3171)
theforeman/foreman-documentation: guides/common/modules/proc_creating-an-image-for-amazon-ec2.adoc

* In the *Username* field, enter the username needed to SSH into the machine.
* In the *Image ID* field, enter the image ID provided by Amazon or an operating system vendor.
ifndef::satellite[]

> This ID can be found within Amazon AWS or on operating system specific pages like https://cloud.debian.org/images/cloud/[debian.org] or https://cloud-images.ubuntu.com/locator/ec2/[ubuntu.com].

Reviewer: ```suggestion
You can find the ID in Amazon AWS or on operating system specific pages such as https://cloud.debian.org/images/cloud/[debian.org] or https://cloud-images.ubuntu.com/locator/ec2/[ubuntu.com].
```
Please change passive voice to active. 
The second suggestion  ("like" to "such as") is purely subjective, not required.

Author: Applied both suggestions.
